### ISSUE TITLE
RouteModel fails to instantiate for departures with non-procedural flightplans

### ISSUE THREAD

**Issue by erikquinn** [openscope/openscope#370]
For aircraft that wish to depart and fly direct to a given fix, the RouteModel fails to instantiate, because it expects all departures to use a standard route (procedure). It must be able to support spawn patterns of `FIXXA..FIXXB..FIXXC`, or even just `FIXXA`.

In particular, this can be seen on @indianbhaji's KPVD pull request, openscope/openscope#101 (f/107).

**Comment by n8rzz** [comment]
I see the issue now.  Its in `AircraftController._setDestinationFromRouteOrProcedure()`.  This method was set up to [eventually] set the `destination` property of an aircraft since it was _always_ used for the `AircraftStripView`.  That is no longer the case, though the `destination` property is used frequently throughout the app.

```javascript
_setDestinationFromRouteOrProcedure({ destination, routeString }) {
    let destinationOrProcedure = destination;

    if (!destination) {
        const routeModel = new RouteModel(routeString);
        destinationOrProcedure = routeModel.procedure;
    }

    return destinationOrProcedure;
}
```

**Comment by erikquinn** [comment]
(unsent from a while ago) Off the top of my head, I only remember the uses of the destination being the strip (which it should not need), the "have I left the airspace properly" check (which should only confirm you're not on a heading), and I think that's it? Really, that can go boom, and at the moment should not be utilized by anything new.

**Comment by erikquinn** [comment]
Closed via #376.
